.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
I am good with PHPCS fixer being run at large to get the baseline style into the repo, but may we have PHPCS evaluation added to the TravisCI config to catch infractions moving forward?

Thanks a lot for this contribution. I have been thinking about this for awhile but never got around to it.